colander_tools/timezone.py

```diff
@@ -0,0 +1,24 @@
+import pytz
+from pytz import exceptions
+
+import colander
+
+
+class TimezoneType(colander.SchemaType):
+    def serialize(self, node, appstruct):
+        if appstruct is colander.null:
+            return colander.null
+
+        return str(appstruct)
+
+    def deserialize(self, node, cstruct):
+        if cstruct is colander.null:
+            return colander.null
+
+        try:
+            return pytz.timezone(cstruct)
+        except exceptions.UnknownTimeZoneError:
+            raise colander.Invalid(
+                node,
+                "`{}` is an invalid timezone".format(cstruct),
+            )
```

Comment:
Like in https://github.com/platformsh/colander-tools/blob/master/colander_tools/strict.py#L22